buildpack/generate.go

```diff
@@ -114,6 +115,7 @@ func readOutputFilesExt(d ExtDescriptor, extOutputDir string, extPlanIn Plan, lo
 	var err error
 	var dfInfo DockerfileInfo
 	var found bool
+	var contexts []ContextInfo
```

Comment:
When we call `runGenerateCmd` we'll also need to set `extOutputDir` correctly:

> the $CNB_OUTPUT_DIR must point to the $CNB_GENERATED_DIR/<extension-id> folder instead of a temporary directory.

Reply:
It's a bit hidden, but should be addressed by https://github.com/buildpacks/lifecycle/blob/a3fc1ef8508c88c1cdc37890b9224096241ebe66/phase/generator.go#L83. WDYT?

Reply:
Sounds good :)